subprojects/core/src/test/groovy/org/gradle/process/internal/JvmOptionsTest.groovy

```diff
@@ -80,21 +80,6 @@ class JvmOptionsTest extends Specification {
         parse("-Xms1G -Dfile.encoding=UTF-8 -Dfoo.encoding=blah -Dfile.encoding=UTF-16").allJvmArgs == ["-Dfoo.encoding=blah", "-Xms1G", "-Dfile.encoding=UTF-16", *localePropertyStrings()]
     }
 
-    def "debug option can be set via allJvmArgs"() {
```

Comment:
❓ If we remove this test, we should remove the `debug` getter/setter as well.

Reply:
As we discussed yesterday, we slightly altered how JavaExec handles debug JVM arguments, hence the removal of the tests. I'm going to add it as a possible breaking change to the upgrade notes.